src/data/eventsourcePolyfill.ts

```diff
@@ -0,0 +1,7 @@
+import {defer, shareReplay} from 'rxjs'
+import {map} from 'rxjs/operators'
+
+export const eventSourcePolyfill = defer(() => import('@sanity/eventsource')).pipe(
```

Comment:
this will lazy load the polyfill, share it between consumers and cache it forever

Reply:
Really cool! And so much more elegant than the old `new Promise` song and dance 🤯